package.json

```diff
@@ -400,6 +400,17 @@
                     "default": true,
                     "description": "Show additional context in hover tooltips when available. This is often the type local variable declaration.",
                     "scope": "resource"
+                },
+                "rust.trace.server": {
+                    "type": "string",
+                    "enum": [
+                        "off",
+                        "messages",
+                        "verbose"
+                    ],
+                    "default": "off",
+                    "description": "Traces the communication between VS Code and the RLS.",
```

Comment:
Minor nit:
```suggestion
                    "description": "Traces the communication between VS Code and the Rust language server.",
```
To keep parity with some of the existing, built-in language servers:
![rust trace](https://user-images.githubusercontent.com/3093213/50691680-c10ff080-1031-11e9-899f-b92124eef40d.png)